### PR TITLE
Two authentication method in one App

### PR DESCRIPTION
**Pull request by mohsspark829**
Thanks:)

### CODE REVIEW

**Comment by mohsspark829** [comment]
Hi Jonas, 

I am getting this error in PR
fix end of files.........................................................Passed
trim trailing whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing docs/docs/multi-tenant/azure_setup.mdx
Fixing docs/docs/single-tenant/azure_setup.mdx

Not sure how to fix it. Could you please advise.

Thanks

**Comment by JonasKs** [comment]
The files have probably been fixed, but you need to `git add` them again, since they've changed. `pre-commit` only runs on staged files (files you've added to the commit), so when you:
- add a file
- run pre-commit
  - pre-commit changes a file
- write git status 

You'll see that the change made by pre-commit isn't staged. 

As a side note: I'm at the airport, going to PyCon and a little vacation, so it might take a while before I get to review this PR 😊

File: docs/docs/multi-tenant/azure_setup.mdx [hunk 1]
```diff
@@ -3,14 +3,20 @@ title: Azure configuration
 sidebar_position: 1
 ---
 
-We'll need to create two application registrations for Azure AD authentication to cover both direct API
-use and usage from the OpenAPI (swagger) documentation.
+We'll need to create two authentication methods for Azure AD authentication to cover both direct API use and usage from the OpenAPI (swagger) documentation. We can achieve this goal in two ways:
+<ol>
+<li> Create two different Azure Apps </li>
+<li> Create one Azure App with two (an App can have mutile authentication method) authentication methods </li>
```
Comment (rhuanbarreto) [inline docs/docs/multi-tenant/azure_setup.mdx:9]:
```suggestion
<li> Create one Azure App with two (an App can have multiple authentication methods) authentication methods </li>
```

**Comment by mohsspark829** [comment]
Thanks alot, no rush at all, Enjoy  your vacation and no work please:)
I will do it through git command on Monday (Friday night in New Zealand).
Today I did it through the github website.

Have a safe trip:)

Cheers
Mohsen

**Comment by JonasKs** [comment]
Hey @mohsspark829 😊 I have not forgot about this, but have not found the time. I'll try to get back to you soon.

**Comment by JonasKs** [comment]
I appreciate your PRs, and I'm sorry I have to close them, but the diff between these and main has diverged too much.

**Pull request closed without merging**